source/AMDSBufferGroupInfo.h

```diff
@@ -43,6 +53,10 @@ class AMDSBufferGroupInfo
 	void setUnits(const QString &units) { units_ = units; }
 	/// Set the axes of the bufferGroupInfo
 	void setAxes(const QList<AMDSAxisInfo> &axes) { axes_ = axes; }
+	/// Set whether the bufferGroup can be flattened
+	void setFlattenEnabled(const bool enabled) { flattenEnabled_ = enabled; }
+	/// Set how the bufferGroup should be flattened
+	void setFlattenMethod(const DataFlattenMethod method) { flattenMethod_ = method; }
```

Comment:
This should be `const DataFlattenMethod &`.  Also, is there a reason why it doesn't have an AMDS prefix?  Seems a little out of place.